src/layouts/LandingPageLayout.res

```diff
@@ -31,33 +31,35 @@ module PlaygroundHero = {
     {
       res: `module Button = {
   @react.component
-  let make = (~count: int) => {
+  let make = (~count) => {
     let times = switch count {
     | 1 => "once"
     | 2 => "twice"
-    | n => Belt.Int.toString(n) ++ " times"
+    | n => n->Int.toString ++ " times"
     }
-    let msg = "Click me " ++ times
 
-    <button> {msg->React.string} </button>
+    <button> {\`Click me $\{times\}\`->React.string} </button>
```

Comment:
Use template literals to showcase the feature

Reply:
Great to use template literals, but maybe we should still leave the `msg` var?

Reply:
I don't really like the `msg` name. And also I prefer putting texts directly to jsx

Reply:
My idea was that keeping it separate introduces the concepts of template literals and `React.string` separately (and also needs less width).

What's your opinion @zth @ryyppy?

Reply:
No clear opinion. I can see both pros and cons to introducing template literals here. 🤷

Reply:
I'd still keep the variable. If you don't like `msg`, you may also go with `message`.

Reply:
The line length is a good argument here. But if we want to return back to a variable, I'd appreciate some naming suggestions besides `msg`.

Reply:
What do you think about `text`?

Reply:
`text` is fine with me!